src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersion.cs

```diff
@@ -11,6 +11,8 @@ namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Model;
 
 public class DataSetVersion : ICreatedUpdatedTimestamps<DateTimeOffset, DateTimeOffset?>
 {
+    public const string FirstVersionString = "1.0";
```

Comment:
Haha.  You know what, it was because I was originally using it in tests, but have since rolled that back out.  I'm just going to return it to the way it was - no const.